.gitmodules

```diff
@@ -38,3 +38,45 @@
 [submodule "third_party/thrust"]
 	path = third_party/thrust
 	url = https://github.com/NVIDIA/thrust
+[submodule "third_party/libsndfile"]
```

Comment:
I am missing information about a tag for a particular release.

Reply:
This is tracked by git. At the very bottom you have a list of relevant sha (git gibmodules don't track tags, just sha). You need to check tags on your own in each repo (git describe --tags in each subrepo).